app/retail/templates/emails/quarterly_stats.html

```diff
@@ -0,0 +1,302 @@
+    color: #0D0764;
+    padding: 1em;
+  }
+
+  .bounty-img {
+    border-radius: 50%;
+    background-color: #DBD0FF;
+    color: #3E00FF;
+    float: left;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .bounty-fund-img {
+    border-radius: 50%;
+    background-color: #D6FBEB;
+    color: #00A55E;
+    float: right;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: -3em;
+    margin-left: -1.3em;
+
+  }
+
+  .stat {
+    display: flex;
+    padding: 0.5em;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #grow-oss {
+    margin-top: 5em;
+  }
+
+  .button-action {
+    margin-bottom: 5em;
+    margin-top: 3em;
+  }
+
+  .percent-content {
+    display: flex;
+    justify-content: center;
+    color: #3E00FF;
+  }
+
+  .button-action > .button {
+    padding: 1.3em;
+  }
+
+  .full-bordered-circle {
+    border-radius: 100%;
+    border-style: solid;
+    border-width: 5px;
+    border-color: #3E00FF;
+  }
+
+  .user-skill-1, .user-skill-5, .user-skill-9 {
+    background-color: rgba(255, 206, 8, .20);
+    color: rgba(255, 206, 8, 1);
+  }
+
+  .user-skill-2, .user-skill-6, .user-skill-10 {
+    background-color: rgba(249, 0, 198, .20);
+    color: rgba(249, 0, 198, 1);
+  }
+
+  .user-skill-3, .user-skill-7, .user-skill-11 {
+    background-color: rgba(142, 42, 90, .20);
+    color: rgba(142, 42, 90, 1);
+  }
+
+  .user-skill-4, .user-skill-8, .user-skill-12 {
+    background-color: rgba(62, 0, 255, .20);
+    color: rgba(62, 0, 255, 1);
+  }
+
+  .user-skill-lang {
+    margin: 0.2em;
+    border-radius: 0.2em;
+    padding: 0.2em;
+  }
+
+  #stat-container {
+    margin-left: 20%;
+    margin-right: 20%;
+    padding: 5em;
+  }
+
+  .bounty-completion-circle {
+    color: #F9006C;
+    border-color: #F9006C;
+  }
+
+  @media (max-width: 600px) {
+    #stat-container {
+      margin-left: 0px !important;
+      margin-right: 0px !important;
+      padding: 0px !important;
+    }
+
+    .stat-box {
+      display: flex;
+      justify-content: center;
+      text-align: center;
+    }
+  }
+</style>
+<div id="stat-container">
+    <h1>GITCOIN QUARTERLY</h1>
+
+    {% if user_active_in_last_quarter %}
+        <h2>You've Been BUILDing</h2>
+        <h4>An overview of your stats from the last 3 months.</h4>
+        <div class="centered-contents stat-box row">
+            <div class="stat-img total-transaction col-sm-12">
+                <p class="content">{{user_total_earned_usd|intcomma}} USD</p>
+            </div>
+            <div class="stat-contents col-sm-12">
+                <p>You earned {{user_total_earned_eth|intcomma}} ETH ({{user_total_earned_usd|intcomma}} USD) from {{user_fulfilled_bounties_count|intcomma}} completed bounties, for an average of {{user_avg_eth_earned_per_bounty|intcomma}} ETH ({{ user_avg_usd_earned_per_bounty|intcomma}} USD) per bounty!</p>
```

Comment:
Typically, you want to use space django template variables like: `{{ example }}`, but obviously {{example}} works.  In any case, we should remain consistent, one way or another.

Ref: `({{ user_avg_usd_earned_per_bounty|intcomma}} USD)`